systemtest/src/test/java/io/strimzi/systemtest/AbstractClusterTest.java

```diff
@@ -152,4 +153,21 @@ String globalVariableJsonPathBuilder(String variable) {
                 .filter(event -> event.getInvolvedObject().getName().equals(resourceName))
                 .collect(Collectors.toList());
     }
+
+    public void sendMessages(String messages, String clusterName, String topic) {
+        LOGGER.info("Sending messages");
+        String command = "echo -e \"" + messages + "\" | sh bin/kafka-console-producer.sh --broker-list " +
+                clusterName + "-kafka:9092 --topic " + topic + " & sleep 20";
```

Comment:
maybe the "sleep" could be a parameter in order to reuse `sendMessages` for tests where we need a different value for it ?